<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTab</class>
 <widget class="QWidget" name="MainTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Top">
   <item>
    <widget class="QLabel" name="waterRemainingLabel">
     <property name="text">
      <string>WATER REMAINING: --- %</string>
     </property>
    </widget>
   </item>
   <item>
    <!-- Main Horizontal Layout (Left: Entry, Right: Numpad) -->
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <!-- Left Side Frame -->
      <widget class="QFrame" name="entryFrame">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="entryFrameLayout" stretch="0,0,1,0">
        <item>
         <!-- Group for Manual Volume Input (This is at the top) -->
         <widget class="QWidget" name="manualVolumeGroup" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>BOTTLE VOLUME:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="manualVolumeLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <!-- Ice Checkbox Container -->
         <widget class="QWidget" name="iceContainerWidget" native="true">
          <layout class="QHBoxLayout" name="iceLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="iceLabel">
             <property name="text">
              <string>BOTTLE HAS ICE?</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="iceSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="iceCheckBox">
             <property name="objectName">
              <string>iceCheckBox</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <!-- Tagged Bottle Info -->
         <widget class="QGroupBox" name="tagInfoGroup">
          <property name="title">
           <string>TAGGED BOTTLE INFO</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>TAG UID:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="tagUidLabel">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>VOL:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="tagVolLabel">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>NAME:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="tagNameLabel">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>EMPTY:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="tagEmptyLabel">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="bottomButtonLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="backButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>&lt; Back</string>
            </property>
            <property name="objectName">
             <string>backButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="bottomSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="startStopButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>START</string>
            </property>
            <property name="objectName">
             <string>startStopButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <!-- Right Side Numpad Frame -->
      <widget class="QFrame" name="numpadFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred"/>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <!-- === NUMPAD GRID (CORRECTED) === -->
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="num7">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
          <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>7</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="num8">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>8</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="num9">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>9</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="num4">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>4</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="num5">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>5</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="num6">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>6</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="num1">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>1</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="num2">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>2</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="num3">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
         <property name="text"><string>3</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="numDot">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>.</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="num0">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>0</string></property><property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="numBackspace">
          <property name="sizePolicy"><sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding"/></property>
           <property name="minimumSize"><size><width>60</width><height>60</height></size></property>
          <property name="text"><string>&lt;-</string></property>
          <property name="class" stdset="0"><string>NumpadButton</string></property>
         </widget>
        </item>
       </layout>
       <!-- === END NUMPAD GRID === -->
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
